webhookValidationConfig.go

```diff
@@ -60,3 +72,30 @@ func (config *ValidatorConfig) BuildURLChecker() (*urlegit.Checker, error) {
 	}
 	return urlegit.New(o...)
 }
+
+// BuildOptions translates the configuration into a list of options to be used to validate the registration
```

Comment:
the logic for the building options should be in ancla because it requires the checker which we build in BuildURLChecker and if we keep this in webhook-schema we will have circular dependency.